pom.xml

```diff
@@ -195,20 +195,9 @@
                   <excludes>
                     <exclude>**/CuFileTest.java</exclude>
                     <exclude>**/CudaFatalTest.java</exclude>
-                    <exclude>**/ColumnViewNonEmptyNullsTest.java</exclude>
```

Comment:
Curious why pom.xml is being changed here?  This is unrelated to xxhash64 changes and effectively reverts #1252.